libra-node/src/main_node.rs

```diff
@@ -287,6 +248,14 @@ pub fn setup_environment(node_config: &mut NodeConfig) -> (AdmissionControlClien
         Arc::clone(&executor),
         &node_config,
     );
+    let admission_control = AdmissionControlRuntime::bootstrap(
+        &node_config,
+        ac_network_sender.unwrap(),
+        ac_network_events,
+        validator_network_provider.is_some(),
+    );
+    let ac_client = admission_control.client.clone();
```

Comment:
no need to have client here. Please remove it and make function just return LibraHandle

Reply:
Why are we dropping ac client completely?